Druid/DRMethodCompilerCompiler.class.st

```diff
@@ -38,6 +38,27 @@ DRMethodCompilerCompiler >> configureForCompilerClass: aCompilerClass [
 		optimisations add: DRCleanControlFlow new. ]
 ]
 
+{ #category : 'accessing' }
+DRMethodCompilerCompiler >> configureForCompilerClassWithoutInline: aCompilerClass [
```

Comment:
Would be nice to not duplicate this method, and see if we can manage the inline optimizations based on a boolean (I'm not sure where)

Reply:
This method is still duplicated